### PR TITLE
Trigger VAL builds in the same way as MicroBuild insertion task...

### PR DESCRIPTION
**Pull request by KevinH-MS**
Using BuildPolicy means the PR will be updated with status and links
from the DDRITs and RPS.

### CODE REVIEW

**Comment by KevinH-MS** [comment]
FYI @jasonmalinowski @tmeschter @cston 
/CC @dotnet/roslyn-infrastructure 

What's the best way to test out changes to the insertion tool?

File: src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionTool.VisualStudioTeamServices.cs [hunk 1]
```diff
@@ -106,83 +108,80 @@ private static GitPullRequest CreatePullRequest(string sourceBranch, string targ
             return newestBuild;
         }
 
-        private static async Task<Build> QueueValidationBuildAsync(string branchName)
+        // Similar to: https://devdiv.visualstudio.com/DevDiv/_git/PostBuildSteps#path=%2Fsrc%2FSubmitPullRequest%2FProgram.cs&version=GBmaster&_a=contents
+        private static async Task QueueBuildPolicy(GitPullRequest pullRequest, string buildPolicy)
         {
-            var buildClient = ProjectCollection.GetClient<BuildHttpClient>();
+            WaitForMergeComplete(pullRequest);
```
Comment (jaredpar) [inline src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionTool.VisualStudioTeamServices.cs]: Seems weird to have a void blocking method inside an async one.

Comment (jmarolf) [reply src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionTool.VisualStudioTeamServices.cs]: Yeah, but there is no async API to get this information

Comment (KevinH-MS) [reply src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionTool.VisualStudioTeamServices.cs]: Not "seems"..."it is"... 😸

**Pull request merged**